p/themes/base-theme/template.css

```diff
@@ -577,17 +577,13 @@ a.btn {
 	transition: max-height .3s linear;
 }
 
-.tree-folder-title {
-	display: block;
-	overflow: hidden;
-	white-space: nowrap;
-	text-overflow: ellipsis;
-}
-
 .tree-folder-title .title {
 	display: inline-block;
 	width: 100%;
 	vertical-align: middle;
+	overflow: hidden;
```

Comment:
Is the hidden necessary for it to work?

Reply:
see the next lines too. It is necessary for the ellipsis.